code/modules/events/holiday/vday.dm

```diff
@@ -47,7 +47,7 @@ GLOBAL_LIST(valentine_mobs)
 
 			forge_valentines_objective(L, date)
 			forge_valentines_objective(date, L)
-		else
+		else if(prob(15)) // 15% of people who didn't get a date will become martyr. this will only be one player per round, so 15% of rounds during valentines
```

Comment:
This calculation is incorrect, it will only apply to 15% of rounds with an odd player count and 0% of rounds with an even player count.

Reply:
Not done with the PR actually, I've doubled the chance now.